karma.conf.js

```diff
@@ -1,29 +1,28 @@
 // Karma configuration
 
 module.exports = function(config) {
-  var customLaunchers = {};
-  var browsers = [];
-  var useSauce = false;
+  const customLaunchers = {};
+  let browsers = [];
+  let useSauce = false;
+  // New firefox versions don't need transpilation but Sauce Labs only support Firefox 45
+  let transpileToES5 = ['internet explorer', 'firefox'].includes(process.env.TEST_BROWSER_NAME);
```

Comment:
I think it's more important to test a current Firefox without transpilation than to test that old version. So let's comment out the Linux tests with a FIXME that we're waiting for Sauce to upgrade.